src/components/AllArticles.jsx

```diff
@@ -1,14 +1,44 @@
 import axios from "axios";
+import React, { useEffect, useState} from "react";
 
 const newsApi = axios.create({
-  baseURL: "https://emiles-nc-news.herokuapp.com",
+  baseURL: "https://emiles-nc-news.herokuapp.com/api",
 });
 
 export const AllArticles = () => {
+  const [articles, setArticles] = useState([])
+  
+useEffect(() => {
+  newsApi.get("/articles").then(({data}) => {
+    //console.log(data.articles);
+     setArticles(data.articles)
+   }).catch((err) => {
+     console.log(err);
+   })
```

Comment:
would be nice to extract your api call into a `api.js` file!

be sure to delete comments & console.logs before commiting too